### ISSUE TITLE
Link to project website on Github About is inactive

### ISSUE THREAD

**Issue by thatlittleboy** [copier-org/copier#657]
Would just like to highlight that the Website in the Github About section points to a dead(?) link.

<img width=375 src="https://user-images.githubusercontent.com/30731072/166402617-8f6429a1-424c-42ff-addb-3ad4f267c023.png">

Seems like the docs website is https://copier.readthedocs.io/en/stable/ instead (and has been for the past couple years).

**Comment by yajo** [comment]
I just tested and the link works. It leads you to the version chooser page:

![imagen](https://user-images.githubusercontent.com/973709/166412361-ce09065d-4806-4d80-9601-f7fbf0f71a09.png)

We found out many people were using v5.x and getting to docs of v6.x, getting confused. And it's even more confusing thanks to https://github.com/mkdocstrings/pytkdocs/issues/122, so I just changed that link.

I'd really love to have a way for it to get the version from a function instead of github api, and to have a version chooser dropdown in the docs.
